tests/containers/single_container_webui.pm

```diff
@@ -6,7 +6,8 @@ sub run {
   my ($self) = @_;
   my $volumes = '-v "/root/data/factory:/data/factory" -v "/root/data/tests:/data/tests" -v "/root/openQA/container/webui/conf:/data/conf:ro"';
   my $certificates = '-v "/root/server.crt:/etc/apache2/ssl.crt/server.crt" -v "/root/server.crt:/etc/apache2/ssl.crt/ca.crt" -v "/root/server.key:/etc/apache2/ssl.key/server.key"';
-
+  
+  assert_script_run("mkdir -p -m a=rwx /root/data/{factory,tests}");
```

Comment:
hm, do we really only need this in tests or do users run into this as well?

Reply:
If they run this using directly without the docker-compose yes, users will have the same problem because the openQA is going to be executed using the geekotest user and the proprietary of these directories inside the container is root

I did a test creating the tests and factory directories as a non-root user
```
drwxr-xr-x  2 suse      suse    64B Jul 19 14:01 factory
drwxr-xr-x  2 suse      suse    64B Jul 19 14:01 tests
```

But inside the container the owner is root, so yes, will be a problem for the user too

```
144255a00bf5:/data # ls -lh
total 0
drwxr-xr-x 2 root root 40 Jul 19 11:57 factory
drwxr-xr-x 2 root root 40 Jul 19 11:57 tests
```

Reply:
My point is: If it's an issue that users could also hit then don't just apply a fix in tests but in the underlying project, i.e. openQA itself